modules/Logging/src/server/ConsoleLogWinstonOverride.coffee

```diff
@@ -1,15 +1,15 @@
 ACAS_HOME="../../.."
 util = require('util')
 winston = require('winston')
-logger = new (winston.Logger)
+logger = winston.createLogger({})
 config = require "#{ACAS_HOME}/conf/compiled/conf.js"
 
 shouldWarn = false
 warningMessage = ""
 
 if config.all.server.log?.level?
 	logLevel = config.all.server.log.level.toLowerCase()
-	allowedLevels = Object.keys(winston.levels)
+	allowedLevels = Object.keys(winston.config.syslog)
```

Comment:
Yes, this was a change in winston.  This is to setup a winston logger instead of just normal console.log messages (which don't have time stamps).  This is someone old though as you get timestamps with running docker and logging to stdout.